flink-runtime/src/main/java/org/apache/flink/runtime/rpc/akka/AkkaRpcServiceUtils.java

```diff
@@ -81,30 +102,46 @@ public static RpcService createRpcService(
 			String hostname,
 			String portRangeDefinition,
 			Configuration configuration) throws Exception {
-		final ActorSystem actorSystem = BootstrapTools.startActorSystem(configuration, hostname, portRangeDefinition, LOG);
-		return createRpcService(configuration, actorSystem);
+		return createRpcService(
+			hostname,
+			portRangeDefinition,
+			configuration,
+			AkkaUtils.getFlinkActorSystemName(),
```

Comment:
Right here we're defining a default logic that is a duplicate of `BootstrapTools.startActorSystem`. Only one method should do this kind of thing.
Basically, just add a new method that calls `BootstrapTools.startActorSystem` without changing other methods.

Reply:
Alright.